dascore/io/rsf/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+RSF format support module.
+
+Notes
+-----
+-
```

Comment:
delete empty section, or add some notes here if you have any. Maybe the version of Madagascar you used to test the output?